### ISSUE TITLE
Does not show the results for each of the source types obtained

### ISSUE THREAD

**Issue by Edd75** [drinfernoo/seren.theme.az2#8]
I know that it is a skin that has little maintenance, but after the last Seren update (v2.2.1), when it shows the results obtained for each type of source, now the blue number for each of them is not shown.

**Comment by drinfernoo** [comment]
Yeah, I was pretty sure that would happen, as there were lots of changes to the way Seren handles windows in 2.2.0.
